include/sys/fs/zfs.h

```diff
@@ -1164,6 +1174,7 @@ typedef struct vdev_stat {
 	uint64_t	vs_read_errors;		/* read errors		*/
 	uint64_t	vs_write_errors;	/* write errors		*/
 	uint64_t	vs_checksum_errors;	/* checksum errors	*/
+	uint64_t	vs_dio_verify_errors;	/* DIO write verify errors */
```

Comment:
This new field needs to be moved to the end of the structure.  The `VDEV_STAT_VALID` guard should also be used in `print_status_config()` to handle the case where new user utilities are used with old kmods.

Reply:
I went ahead and updated this.